index.ts

```diff
@@ -21,9 +21,14 @@ export async function main() {
     const cluster = (process.env.CLUSTER || 'localnet') as Cluster;
 
     const programNameToId = getProgramMap(cluster);
-    const endpoint = process.env.ENDPOINT_URL || 'http://127.0.0.1:8899';
+    const endpoint = process.env.ENDPOINT_URL || 'http://localhost:8899';
```

Comment:
btw, not quite sure if localhost will work for solana 14+. I vaguely remember 0.0.0.0 worked

Reply:
I tested it, it should be working.